### PR TITLE
Add network metrics endpoint

### LINKED ISSUE #2140: Add network metrics endpoint
https://github.com/iotaledger/iota.go/pull/703

### PR DESCRIPTION
**Pull request by thibault-martinez**
Fixes https://github.com/iotaledger/iota-sdk/issues/2140

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

File: .github/actions/private-tangle/setup/action.yml [hunk 1]
```diff
@@ -40,6 +38,6 @@ runs:
     - name: Wait for tangle to start
       shell: bash
       run: wget -qO- https://raw.githubusercontent.com/eficode/wait-for/v2.2.4/wait-for | sh -s -- -t 120 http://localhost:8050/health -- echo "Tangle is up"
-    - name: Wait for faucet to start
-      shell: bash
-      run: wget -qO- https://raw.githubusercontent.com/eficode/wait-for/v2.2.4/wait-for | sh -s -- -t 120 http://localhost:8088/health -- echo "Faucet is up"
+    # - name: Wait for faucet to start
```
Comment (DaughterOfMars) [inline .github/actions/private-tangle/setup/action.yml:42]: ?

Comment (Thoralf-M) [reply .github/actions/private-tangle/setup/action.yml:42]: Faucet is currently broken as it's not update to the latest iota-core API changes (the ones changed in this PR :trollface: )

Comment (DaughterOfMars) [reply .github/actions/private-tangle/setup/action.yml:42]: Add a todo then?

Comment (thibault-martinez) [reply .github/actions/private-tangle/setup/action.yml:42]: done

**Review by DaughterOfMars — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Comment by Thoralf-M** [comment]
Not sure if we want to wait with merging, since it would break with the alphanet

**Comment by begonaalvarezd** [comment]
> Not sure if we want to wait with merging, since it would break with the alphanet

maybe lets wait until the next alphanet update, we could use in the meantime some PRs you incorporate that are not breaking

**Pull request merged**